website/language.py

```diff
@@ -108,6 +108,11 @@
 
 </ul>
 '''
+REGISTRATION_EMBARGO_INFO = '''
```

Comment:
You can choose to make your registration public, private, or embargoed. An embargoed registration is private for a predetermined amount of time and automatically made public when the time of embargo expires. Once a registration is made public, it cannot be made private again unless the registration is retracted. 

If you chose to embargo your registration, a notification email will be sent to (all other contributors or contributors with administrative permissions?) on this project. Any disapproval will prevent the registration from entering embargo, and the registration will remain private. After X hours, if there have been no disapprovals, the registration will enter the embargo period.